pages/events.js

```diff
@@ -23,6 +23,39 @@ import { compact } from 'lodash'
 import theme from '@hackclub/theme'
 
 const events = [
+  {
+    name: 'Apocalypse',
+    description: `The high school hackathon where you build fun tech to survive the zombie apocalypse!`,
+    logo: 'https://cloud-7yl2xnwax-hack-club-bot.vercel.app/0image.png',
+    location: 'Toronto, Canada',
+    season: 'Spring',
+    year: '2024',
+    // repo: 'apocalypse',
+    image: '',
+    link: 'https://apocalypse.hackclub.com'
+  },
+  {
+    name: 'Wonderland',
+    description: `🕳️🐇 How would you and your friends use a 🥕carrot in a hackathon project? From Alice in Wonderland themed mystery chests, come the infinities of Wonderland.`,
+    logo: '',
+    location: 'Boston, Massachusetts',
+    season: 'Winter',
+    year: '2024',
+    // repo: 'wonderland',
+    video: 'https://www.youtube.com/embed/X1EVT0vUxAg',
+    link: 'https://wonderland.hackclub.com'
+  },
+  {
+    name: 'Leaders Summit',
+    description: `A weekend of invention, collaboration, and friendship in San Francisco with 50 Hack Club leaders from around the world.`,
+    logo: '',
```

Comment:
you can put https://cloud-okxvflzju-hack-club-bot.vercel.app/1summit2.png for the summit logo!

https://cloud-okxvflzju-hack-club-bot.vercel.app/0summit1.png or https://cloud-okxvflzju-hack-club-bot.vercel.app/2summit3.png also works I think; I just took the favicon